sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -15,8 +15,10 @@
 - Renamed `GetShareStatisticsResult` to `ShareStatistics`.
 - Renamed `GetShareDirectoryPropertiesResult` to `ShareDirectoryProperties`.
 - Renamed `GetShareFilePropertiesResult` to `ShareFileServiceProperties`.
```

Comment:
Shouldn't `GetProperties` return a `ShareServiceProperties` in this case (similar to `BlobServiceProperties`). Remove the `File` in the name (in source too)?

Reply:
It should be `FileServiceProperties` right?

Reply:
File service <-> Blob service
container <-> share

Reply:
`FileServiceProperties` makes sense.

Looking at the other ones like `DataLake`, should we drop the `DataLake` prefix?
Similarly, for `ShareDirectoryProperties`, drop the `Share`?

Reply:
`DataLake` and `Share` prefixes are already dropped, aren't they?

Reply:
Just verified, they are. All's good.